packages/api/AlvTime.Persistence/Repositories/AlvEconomyData/OvertimePayoutStorage.cs

```diff
@@ -0,0 +1,117 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using AlvTime.Business.EconomyData;
+using AlvTime.Business.FlexiHours;
+using AlvTime.Persistence.EconomyDataDBModels;
+using FluentValidation;
+
+namespace AlvTime.Persistence.Repositories.AlvEconomyData
+{
+    public class OvertimePayoutStorage : IOvertimePayoutStorage
+    {
+        private readonly AlvEconomyDataContext _economyContext;
+        private readonly IEmployeeHourlySalaryStorage _employeeHourlySalaryStorage;
+
+        public OvertimePayoutStorage(AlvEconomyDataContext economyContext, IEmployeeHourlySalaryStorage employeeHourlySalaryStorage)
+        {
+            _economyContext = economyContext;
+            _employeeHourlySalaryStorage = employeeHourlySalaryStorage;
+        }
+
+        public OvertimePayoutResponsDto DeleteOvertimePayout(int userId, DateTime date)
+        {
+            var overtimePayout =
+                _economyContext.OvertimePayouts.FirstOrDefault(op => op.UserId == userId && op.Date.Equals(date));
```

Comment:
Lurer på om det kanskje finnes en bedre måte å matche Payout på enn å sjekke dato. Ser for meg at det kan bli litt krøll med tidspunkt der. Er det mulig å innføre en slags Id i OvertimePayouts som matche Id til PaidOvertime i den andre contexten? Alternativt kunne man flyttet hele PaidOvertime tabellen fra Alvtime_dbContext til AlvEconomyDataContext, men det blir kanskje mye styr...

Reply:
Jeg er enig, skal sjekke. (Mulig jeg kan bruke Id'en fra paid overtime?)

Reply:
Byttet det ut med Iden fra PaidOvertime. Da skal det i teorien bli helt riktig